cadcdata/cadcdata/storageinv.py

```diff
@@ -588,15 +651,15 @@ def cadcinfo(self, id):
         logger.debug('File info: {}'.format(file_info))
         return file_info
 
-    def _get_transfer_urls(self, id, is_get=True):
+    def _get_transfer_urls(self, id, params=None, is_get=True):
         if not self.transfer:
             # this is site location
             return ['{}/{}'.format(self.files, id)]
         trans = net.Transfer(self._cadc_client._get_session(), )
         return trans.transfer(
             endpoint_url=self.transfer, uri=id,
             direction='pullFromVoSpace' if is_get else 'pushToVoSpace',
-            with_uws_job=False)
+            with_uws_job=False, cutout=params)
```

Comment:
If the `params` are `{'META': 'true'}`, is that just a special form of cutout?

Reply:
No. I think that cutouts always return some bytes.